openstack/nova/files/compute.nova.conf

```diff
@@ -117,18 +112,24 @@ user_domain_id = default
 auth_strategy = keystone
 admin_tenant_name = service
 admin_username = neutron
-admin_password = {{salt['pillar.get']('virl:password', salt['grains.get']('password', 'password')) }}
+admin_password = {{ virl.ospassword }}
 admin_auth_url = http://{{ virl.controller_ip }}:35357/{{ virl.keystone_auth_version }}
 {% endif %}
 
+[vnc]
+vncserver_listen = {{ virl.int_ip }}
+vncserver_proxyclient_address = {{ virl.int_ip }}
+novncproxy_port = {{ virl.vnc_port }}
+novncproxy_base_url = http://127.0.1.1:{{ virl.vnc_port }}/vnc_auto.html
```

Comment:
It is OK that this is localhost - STD knows it should pass in the controller_ip or virl_local_ip instead.